### DISCUSSION TITLE
2022/03/IPFS和博客/

### DISCUSSION THREAD

**Discussion by SouthFox-D** [SouthFox-D/SouthFox-D.github.io#60] [博客评论]
https://blog.southfox.me/2022/03/IPFS和博客/

**Comment by ggq89** [comment]
厉害！，你这是放在ipfs上的静态网站吗？！

**Reply by SouthFox-D** [reply]
是静态网站，但放没放在 `ipfs` 上要看是什么访问方式访问，没通过 `ipfs` 网关访问的那就没有放在 `ipfs` 上。

**Comment by l3ony2k** [comment]
太酷了！Mark 一下将来学习。另外您的博客也好酷，超有风格 ☺️
